src/main/kotlin/lotto/business/PrizeResults.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.business
+
+class PrizeResults(prizeCountMap: Map<LotteryPrize, Int>) {
+
+    private val _prizeCountMap = prizeCountMap.toMutableMap()
+
+    val prizeCountMap: Map<LotteryPrize, Int>
+        get() {
+            return LotteryPrize.values().associateWith {
+                _prizeCountMap.getOrDefault(it, 0)
```

Comment:
아래와 같은 방식으로도 쓸 수 있어요. 😃
```suggestion
               _prizeCountMap[it] ?: 0
```